src/main/java/com/minecolonies/coremod/items/ItemAncientTome.java

```diff
@@ -0,0 +1,21 @@
+package com.minecolonies.coremod.items;
+
+import com.minecolonies.coremod.creativetab.ModCreativeTabs;
+
+/**
+ * Class handling the AncientTome item.
+ */
+public class ItemAncientTome extends AbstractItemMinecolonies
```

Comment:
I would make the ancient tome glow (like an enchantment) if a Raid is going to happen in the next night!
Make the tome useffull in my opinion.